scripts/build-doc

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+/* eslint-env node */
+
+require('./util/work-from-root')();
+
+var docBuilder = require('./util/doc-builder')({
```

Comment:
we support 0.12, but we're considering bumping to the LTS (4.4.7). When we do that, we'll adjust.